### DISCUSSION TITLE
Webcam/Camera issue maybe after updating window

### DISCUSSION THREAD

**Discussion by catunderrain** [community/community#132053] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When I use the Camera app, cam on Messenger, or cam for Python CV2, it doesn't work and throws back a message like: No Camera Attached. This happened quite after I updated something new on Windows Update yesterday, and I found my friend had the same issue. 
### This is how the issue on the app Camera:
![image](https://github.com/user-attachments/assets/eea01ddc-1886-4016-a355-9f1e45b985f9)

## This is may when I run python:
```
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
W0000 00:00:1720865808.944165   22516 inference_feedback_manager.cc:114] Feedback manager requires a model with a single signature inference. Disabling support for feedback tensors.
W0000 00:00:1720865808.961879   22516 inference_feedback_manager.cc:114] Feedback manager requires a model with a single signature inference. Disabling support for feedback tensors.
```
## This is the Intergrated Camera that was hidden in Device Manager:

![image](https://github.com/user-attachments/assets/a3e5d66e-b740-4ec5-ad04-ca5a66a0d52b)
#
## This is processes running in my Task Manager:
![image](https://github.com/user-attachments/assets/fd82e43f-3904-4b4d-a026-b882a7aa24ea)

## This is the nearest update history:
![image](https://github.com/user-attachments/assets/29e6afcf-5144-4da0-99b9-bb0d5e142ec5)
#
**If you guys have any order information for this issue, please feel free to ask me. I hope to receive your solutions soon!**

**Comment by ghost** [comment]
Hi, this is most likely a bug of Windows itself, as you said, it appeared after the update, so it’s possible. Try rolling back this update

**Reply by catunderrain** [reply, marked as answer]
Thanks for your suggestion, but i can't find any rollback for the camera driver. Regarding Window Update, I think I should choose to find out more information about bug fixes and wait for a patch if the problem really lies in the most recent update, instead of recovering as lots of my work can be lost in the process. Anyway thank you very much for your suggestion!
